pkg/apis/resource/validation/validation.go

```diff
@@ -185,8 +194,8 @@ func ValidateClaimStatusUpdate(resourceClaim, oldClaim *resource.ResourceClaim)
 func validateAllocationResult(allocation *resource.AllocationResult, fldPath *field.Path) field.ErrorList {
 	var allErrs field.ErrorList
 	if allocation != nil {
-		if len(allocation.ResourceHandle) > resource.ResourceHandleMaxSize {
-			allErrs = append(allErrs, field.TooLongMaxLength(fldPath.Child("resourceHandle"), len(allocation.ResourceHandle), resource.ResourceHandleMaxSize))
+		if len(allocation.Context) > 0 {
+			allErrs = append(allErrs, validateResourceHandles(allocation.Context, resource.AllocationResultContextMaxSize, fldPath.Child("context"))...)
```

Comment:
unclear why AllocationResultContextMaxSize is an argument here?

Reply:
This mimics the rest of the code in this file which does this similarly. It was kept this way for consistency.